jdbc/src/test/java/org/apache/metamodel/jdbc/JdbcDataContextTest.java

```diff
@@ -261,6 +265,20 @@ public void testWhereScalarFunction() throws Exception {
         }
     }
 
+    public void testUnsupportedAggregateFunction() throws Exception {
+        final Connection connection = getTestDbConnection();
+        final JdbcDataContext dataContext = new JdbcDataContext(connection);
+        try {
+            dataContext.query().from("customers").select(FunctionType.RANDOM, "customernumber").execute();
+            fail("Exception expected");
+        } catch (MetaModelException e) {
+            assertEquals(
+                    "Aggregate function 'RANDOM' is not supported on this JDBC database. Query rejected: "
+                            + "SELECT RANDOM(\"CUSTOMERS\".\"CUSTOMERNUMBER\") FROM PUBLIC.\"CUSTOMERS\"",
+                    e.getMessage());
+        }
+    }
+
```

Comment:
This is fine, though reformatting the whole file for adding a single function is a pretty history-destructive.

That much of the formatting is pretty weird doesn't make it better :)